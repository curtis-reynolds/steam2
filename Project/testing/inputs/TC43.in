login
admin
addcredit
nonadmin
1001
