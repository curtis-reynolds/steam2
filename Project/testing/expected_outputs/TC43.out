Login successful.
Error: Maximum $1000.00 can be added to an account in a given session.
